### ISSUE TITLE
JavaDecaf does not accept names with spaces

### ISSUE THREAD

**Issue by sergutsan** [SophieKoonin/JavaDecaf#14]
This is fine, but the error should be more explicit. 

Instead of: 

```
JavaDecaf: Compiling file Fact fact.jdc...
Fact fact.java:4: error: '{' expected 
public class Fact fact { 
                 ^ 
1 error
JavaDecaf: Compilation finished in 25 ms
```

the error could be: 

```
JavaDecaf: Compiling file Fact fact.jdc...
File names in JavaDecaf must not contains spaces. Please rename the file.
JavaDecaf: Compilation finished in 25 ms
```

**Comment by sophiekoonin** [comment]
This can be implemented by checking the filename when it's first declared - line 60 of JavaDecafCompiler.java.

**Comment by sergutsan** [comment]
Thanks for the pointer. 

Fixed in 3f5c0471aa52.
